src/tests/functional/shared_test_classes/src/single_op/gather.cpp

```diff
@@ -244,5 +244,77 @@ void Gather8withIndicesDataLayerTest::SetUp() {
     function = std::make_shared<ov::Model>(result, ov::ParameterVector{param}, "gather");
 }
 
+// Gather String support
+std::string GatherStringWithIndicesDataLayerTest::getTestCaseName(const testing::TestParamInfo<gatherStringParamsTuple>& obj) {
+    gatherStringParamsTuple basicParams = obj.param;
+    std::vector<int64_t> indicesData;
+    std::vector<std::string> str_data;
+
+    std::tuple<int, int> axis_batch_idx;
+    std::vector<int> indices;
+    ov::Shape indices_shape;
+    std::vector<InputShape> shapes;
+    ov::element::Type model_type;
+    std::string device_name;
+    std::tie(shapes, indices_shape, axis_batch_idx, model_type, device_name, indicesData, str_data) = basicParams;
```

Comment:
`std::tie` here implies unnecessary copy of dynamically allocated containers which aren't then modified. `const &` per each tuple element is a better choice, although less convenient. This is a common practice in tests, so no need to change - just a comment.